src/app/components/login/login.component.ts

```diff
@@ -31,17 +31,19 @@ export class LoginComponent {
   }
 
   login() {
-     console.log(this.loginForm.value)
+     console.log( ' Login Form Values:',this.loginForm.value)
+
     if (this.loginForm.valid) {
       const loginRequest: LoginRequest = {
         email: this.loginForm.get('email')?.value,
         password: this.loginForm .get('password')?.value
```

Comment:
Can you have a custom method to handle this to avoid repetition e.g

public formField(field: string) {
return this.loginForm.get(field) as FormControl;
}

Reply:
code has been taken off now using the store to handle login